psgallery_status.md

```diff
@@ -1,6 +1,14 @@
 
 PowerShell Gallery Status
 =========================
+### 03/22/2020 02:30 UTC The [PowerShell Gallery](https://powershellgallery.com) was unavailable via HTTPS.
```

Comment:
I suspect this date might not be accurate....

Reply:
fixed.